internal/test/e2e/setup.go

```diff
@@ -168,6 +168,11 @@ func (e *E2ESession) setup(regex string) error {
 		return err
 	}
 
+	err = e.setupVsphereMultiTemplateUbuntu127(regex)
```

Comment:
It might be best to avoid having to do this logic based on test name rather than just being able to use a 127 template. We have to use a no snapshots one?

Reply:
It doesn't have to be a no snapshot one, but need to be a different one then default 1.27 ubuntu

Reply:
So why do we need to call the setup here if we only need to env var for the test?